Formula/h/hyfetch.rb

```diff
@@ -12,10 +12,15 @@ class Hyfetch < Formula
     sha256 cellar: :any_skip_relocation, all: "55446c64fe707ad34df6df6193ccaac2e599597bf6e340a8753881a22f431867"
   end
 
+  depends_on "rust" => :build
   depends_on "python@3.13"
 
   def install
+    # Install to `buildpath` first or else `virtualenv_install_with_resources` will overwrite it.
+    system "cargo", "install", *std_cargo_args(path: "crates/hyfetch", root: buildpath)
     virtualenv_install_with_resources
+    # Install the rust executable where the Python package expects it.
+    (libexec/Language::Python.site_packages("python3")/"hyfetch/rust").install "bin/hyfetch"
```

Comment:
Can we just drop the Python code/dependency given it seems to be a fallback for new rust implementation? If so, should also use GitHub source rather than PyPI.

Will need to handle undoing `all` bottle too.